samples/contrib/versioned-pipeline-ci-samples/helloworld-ci-sample/pipeline.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import kfp.dsl as dsl
+from kfp.gcp import use_gcp_secret
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--commit_id', help='Commit Id', type=str)
```

Comment:
Maybe elaborate more in the `help` field?

Reply:
ok